lib/sqlalchemy/orm/strategies.py

```diff
@@ -1707,7 +1707,7 @@ def _generate_row_adapter(
         context.create_eager_joins.append(
             (
                 self._create_eager_join,
-                context,
+                None,
```

Comment:
can this data member be removed entirely

Reply:
Let me remove that